Assets/Scripts/InputManager.cs

```diff
@@ -53,6 +53,10 @@ private void OnMove(InputValue inputValue)
         var data = inputValue.Get<Vector2>();
         EventManager.TriggerEvent("OnMove", data);
     }
+    private void OnDash(InputValue inputValue)
+    {
+        EventManager.TriggerEvent("OnDash", inputValue.isPressed);

```

Comment:
> If you have some time to create a constant in [Constants.cs](https://github.com/Taliayaya/Project-86/blob/main/Assets/Scripts/Constants.cs) instead of hardcoding the string, it could be great

Same recommendation

Reply:
Oh I see, they just haven't been dealt with yet, ill move more over when I can